src/js/components/Graphs/Legend/legend.spec.jsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import { createLegend } from './legendHelper';
+import { legendArraysMock } from './legend.mock';
+import {
+  LegendHeader,
+  LegendItemIcon,
+  SquareIcon,
+  CircleIcon,
+  RectangleIcon,
+  LineIcon,
+  LegendItemDescription,
+  LegendItem,
+} from './legend.style';
+
+describe('Correctly render legend component', () => {
```

Comment:
I'm lost here, you are saying this should correctly render legend component, but you are not testing Legend component but you build it yourself